source/MRMesh/MRToolPath.h

```diff
@@ -99,6 +99,10 @@ MRMESH_API ToolPathResult constantZToolPath( const Mesh& mesh, const ToolPathPar
 // mesh can be transformed using xf parameter
 MRMESH_API ToolPathResult lacingToolPath( const Mesh& mesh, const ToolPathParams& params, const AffineXf3f* xf );
 
+// compute path of the milling tool for the given mesh with parameters ( direction of milling is from up to down along Z-direction )
+// mesh can be transformed using xf parameter
```

Comment:
all 3 functions have the same comment, it is not good
Please write distinct comments for the user to understand which one to select